pkg/operator/controller/controller.go

```diff
@@ -186,6 +186,12 @@ func (r *ReconcileCDI) Reconcile(request reconcile.Request) (reconcile.Result, e
 		return reconcile.Result{}, err
 	}
 
+	if configMap != nil && configMap.DeletionTimestamp != nil {
```

Comment:
I honestly don't think this will be helpful.  I don't think there is any way the configmap will have a deletion timestamp and the CDI CR will not.  And you will not get here if the CDI CR deletion timestamp is not ini